tests/topotests/bgp_set_aspath_exclude/r1/bgpd.conf

```diff
@@ -9,9 +9,14 @@ router bgp 65001
 !
 ip prefix-list p1 seq 5 permit 172.16.255.31/32
 !
+bgp as-path access-list FIRST permit 2 
+!
 route-map r2 permit 10
- match ip address prefix-list p1
- set as-path exclude 65003
-route-map r2 permit 20
- set as-path exclude all
+ set as-path exclude access-list FIRST
+
+!route-map r2 permit 10
+! match ip address prefix-list p1
+! set as-path exclude 65003
+!route-map r2 permit 20
```

Comment:
do not modify the topotest in that commit, please. there is a separate commit for that.